src/app/containers/ConsentBanner/Banner/index.canonical.jsx

```diff
@@ -42,6 +38,8 @@ const CanonicalConsentBannerContainer = ({ type, onReject, onAccept }) => {
     type === 'cookie'
       ? translations.consentBanner.cookie.canonical
       : translations.consentBanner[type];
+  const ukText = consentBannerConfig.description.uk;
```

Comment:
IMO it's better to write code that is clear rather than write code that is hard to read and leave comments that end up getting out of date when code changes that can be misleading about the code's behaviour and ultimately waste a developer's time.

Maybe better named variables is the option here? What do you think?

Reply:
Thanks for the rec. I can't think of a suitable alternative that would give much more information to the user so happy to keep as is. @andrewscfc gave a good description of `ukText` and `internationalText` last week when reviewing and I was seeking to capture some of that but think it would go against clean code principles